src/Client/Connection.cpp

```diff
@@ -575,6 +577,22 @@ void Connection::receiveHello(const Poco::Timespan & handshake_timeout)
             readIntBinary(read_nonce, *in);
             nonce.emplace(read_nonce);
         }
+        if (server_revision >= DBMS_MIN_REVISION_WITH_SERVER_HELLO_EXTRAS)
+        {
+            UInt64 extras_size;
+            readVarUInt(extras_size, *in);
```

Comment:
Frankly speaking I do not see the point of introducing `hello extras` instead of just sending the settings under only `DBMS_MIN_REVISION_WITH_SERVER_SETTINGS`, is it just for the future if we'd want to add something else?

Reply:
Ok, seems it would be good to just add these words from your comment above to a comment in code:
> Otherwise adding fields to Hello message defies the purpose of chunked protocol (ReadBufferFromPocoSocketChunked et al), IIUC. Proxy would have to be updated to recognize every new field, unless there's some predetermined way to skip all fields.